articles/active-directory/develop/access-tokens.md

```diff
@@ -242,45 +242,34 @@ If the application has custom signing keys as a result of using the [claims-mapp
 
 ### Claims based authorization
 
-The business logic of the application dictates claims based authorization. Some common authorization methods are listed below.
+The business logic of an application determines how authorization should be handled. The general approach to authorization based on token claims, and which claims should be used, is described below.
 
-#### Validate the token
+After a token is validated with the correct `aud` claim, the token tenant, subject, actor must be authorized.
```

Comment:
Potential point of ambiguity here for the uninitiated reader: 
- What does it mean to be "the correct `aud` claim"?

For our purposes, this means: "validate that the `aud` claim matches the app id of your service" - maybe we should spell this out explicitly? (Perhaps this is covered elsewhere in the doc and I just don't see it as part of the PR)

Reply:
Ya, it is explained in the preceding section about validating tokens, but doesn't hurt to remind